writer_downgrade_test.go

```diff
@@ -77,10 +77,15 @@ func (s *WriterDowngradeSuite) TestWriteObjectAndDowngrade() {
 		Trust    bool   `rsf:"trust"` // Not in original struct
 	}
 	// Add a new type that was also not included in the original data.
+	type variation struct {
+		Id          int8   `rsf:"id,skip"`
```

Comment:
Does the field that is indexed on have to be `skip`ped or is that just used for this test case?

Reply:
It doesn't need to be. I've typically skipped them for deduplication purposes since the index will have exactly the same data.